README.md

```diff
@@ -2,10 +2,69 @@
 
 Template from [here](https://github.com/tailwindtoolbox/Landing-Page/blob/master/index.html).
 
-## Usage
+## Requirements
 
-**Assumes you have Node installed. You can get it for your machine [here](https://nodejs.org/en/download)**
+For our developer setup we have assumed Ubuntu 22.04 as our base OS.
```

Comment:
We will not be making this assumption. 90% of developers use Windows or Mac as their daily driver.